### PR TITLE
I39 create django automated tests for event endpoint

### LINKED ISSUE #39: Create Django Automated Tests for Event endpoint
## Basic Information
Verify that each of the CRUD operations work when you have a client that does HTTP Request for the event endpoint

Helpful link: https://www.django-rest-framework.org/api-guide/testing/

You will be adding your test in `/event/tests` folder. You could name create as many files as you want. However prefix the file with `test_` and make sure that functions you create will also have prefix `test_`

### PR DESCRIPTION
**Pull request by wuryantoAji**
## Change Summary
Add unit tests for event endpoint

### Change Form
no documentation needed for unit test

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [x] The change has tests
- [ ] The change has documentation

## Other Information
-

# Related Issue

- Resolve #39

### CODE REVIEW

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Hey It's looking good. However, your branch is not up to date with main - can you please merge main into your branch because team_id has recently been added to the events model :)

File: server/api/event/tests.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from .models import Event
+
+import datetime
+
+
+class EventTests(APITestCase):
+    def setUp(self):
```
Comment (ChantelleKerr) [inline server/api/event/tests.py:11]: You'll also have to include the team FK in your tests

Comment (wuryantoAji) [reply server/api/event/tests.py:11]: oops, I forgot to do that, will fix thanks chantelle

**Review by ChantelleKerr — APPROVED** [review]
Great work! Looks good. Approved.

You can merge now :)

**Pull request merged**